Doppler.HelloMicroservice/appsettings.json

```diff
@@ -1,14 +1,15 @@
 {
-  "Logging": {
-    "LogLevel": {
+  "LogglyConfig": {
+    "CustomerToken": "REPLACE_WITH_CUSTOMER_TOKEN"
+  },
+  "Serilog": {
+    "MinimumLevel": {
       "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
+      "Override": {
+        "Microsoft": "Warning",
+        "Microsoft.Hosting.Lifetime": "Information"
+      }
     }
   },
-  "AllowedHosts": "*",
-  "DopplerSecurity": {
-    "PublicKeysFolder": "public-keys",
-    "PublicKeysFilenameRegex": "\\.xml$"
-  }
```

Comment:
Why is removed this?
I only see that is not related to the title of this PR.

Reply:
I must have deleted it within the conflict solving. I'll fix it